test/integrations/destinations/fb_custom_audience/dataDelivery/other.ts

```diff
@@ -0,0 +1,68 @@
+import { generateMetadata, generateProxyV1Payload } from '../../../testUtils';
+import { ProxyV1TestData } from '../../../testTypes';
+import { getEndPoint } from '../../../../../src/v0/destinations/fb_custom_audience/config';
+
+export const otherScenariosV1: ProxyV1TestData[] = [
+  {
+    id: 'fbca_v1_other_scenario_1',
+    name: 'fb_custom_audience',
+    description: 'user update request is throttled due to too many calls to the ad account',
+    successCriteria:
+      'Should return 429 with message there have been too many calls to this ad-account',
+    scenario: 'Framework',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload({
+          method: 'DELETE',
+          endpoint: getEndPoint('aud1'),
+          headers: {
+            'test-dest-response-key': 'tooManyCallsError',
+          },
+          params: {
+            access_token: 'ABC',
+            payload: {
+              is_raw: true,
+              data_source: {
+                sub_type: 'ANYTHING',
+              },
+              schema: ['DOBY', 'PHONE', 'GEN', 'FI', 'MADID', 'ZIP', 'ST', 'COUNTRY'],
+              data: [['2013', '@09432457768', 'f', 'Ms.', 'ABC', 'ZIP ', '123abc ', 'IN']],
+            },
+          },
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message: 'There have been too many calls to this ad-account.',
+            statTags: {
```

Comment:
can we use common statTags and override only fields that are changed for this test case using { ... }